frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.css

```diff
@@ -5,12 +5,61 @@
 
 .diagrams {
   height: calc(80vh - 170px);
-  width: 100%;
   word-wrap: break-word;
 }
 
+.diagram-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: calc(100% / 3 - 2 * 10px - 2 * 16px); /* 3 rows - margin - padding */
```

Comment:
I would expect the parent node to take care of this. As long as `justify-content` is set correctly.

Reply:
How would you do it? Setting the number of rows is not an option for flexboxes afaik.

Reply:
It should work if you set `flex-basis: 30%;` (little less that 33%, to account for margins).

Reply:
Good improvement. I've changed it for loader & the diagram cards